### DISCUSSION TITLE
Z-Axis mods

### DISCUSSION THREAD

**Discussion by SteveBz** [1coderookie/KobraGoNeoInsights#4] [Show and tell]
I've done both the mods you mention to the z-axis, both the stabiliser and the second z-axis.  I Still need to retichten the off-centre wheels!  Here's some photos:

![WhatsApp Image 2023-03-04 at 16 07 56](https://user-images.githubusercontent.com/1646078/222966974-539a1ce5-1e87-48cb-be7f-f68360e9719a.jpg)
![WhatsApp Image 2023-03-05 at 14 38 50](https://user-images.githubusercontent.com/1646078/222967206-8127d3d8-fe22-4184-b583-7bc0670481eb.jpg)
![WhatsApp Image 2023-03-05 at 14 38 40](https://user-images.githubusercontent.com/1646078/222967212-dacc10cb-2ddf-43dc-9c5f-5ff4f279677d.jpg)
![WhatsApp Image 2023-03-05 at 14 38 27](https://user-images.githubusercontent.com/1646078/222967215-ae83f49b-07f4-4257-9d5e-abfced28bdf2.jpg)

**Comment by 1coderookie** [comment]
Looks good, but I personally would throw out the bearing blocks at the top as I described at the page and tried to explain why to avoid using them.
Did you just use a y-cable w/o adjust the current for the z-axis stepper driver?

**Comment by SteveBz** [comment]
There is a perfect little screw hole near the motherboard for an M3 screw to fit this stepper splitter board.  Then I spliced (soldered) 2 of the extenders together because one isn't quite long enough. Then The Vref for mine was about 0,82v, but others have found .71v, so I guess .7-.8v.  I upped the Vref potentiometer to 1.16v and it works beautifully.

![WhatsApp Image 2023-03-05 at 15 08 33](https://user-images.githubusercontent.com/1646078/222968811-bfd40b7a-81ee-4de8-91d4-bdc42ec28bdc.jpg)
![WhatsApp Image 2023-03-05 at 15 08 20](https://user-images.githubusercontent.com/1646078/222968806-9608f037-2b71-4ad5-99f3-ad33e013d1d9.jpg)

**Comment by SteveBz** [comment]
Here's the one I did, with my comments:

https://www.printables.com/model/364889-anycubic-kobra-neogo-dual-z-mod

**Comment by 1coderookie** [comment]
Ah ok, you used that kind of splitter - great solution mounting it next to the mainboard! If you ever access the mb in the future, could you maybe take a picture of the assembled setup and put it in here as well? I'll link to this discussion in that section of the page, so people can see it actually.
Thanks!

**Comment by SteveBz** [comment]
**Here the original Z-axis cable with black filaments is folded back to the splitter board**

![WhatsApp Image 2023-03-06 at 20 37 54](https://user-images.githubusercontent.com/1646078/223228969-4ea21a9b-4875-4c7e-ac3c-e49122ef4d6e.jpg)

**Here the new multicolour cable goes from the Z-axis driver to the single 'in' port of the splitter board,  I've had to extend it because 1 cable wasn't long enough (it's soldered under the insulation tape!!)**

![WhatsApp Image 2023-03-06 at 20 38 50](https://user-images.githubusercontent.com/1646078/223228970-23d9875e-8008-4e94-91d9-79bee478010c.jpg)

**Here is the whole setup.  Again the stepper cable had the wrong plug and had to be soldered.  Luckily I bought a pack of 5 board with 5 cables, so It didn't matter that I cannibalised a few.  NB The second motor was anti-parallel to the original, so I switched the left-hand two filaments when soldering.  That fixed it.  (Left-had two or right-hand two, doesn't matter, just not the middle two).**

![WhatsApp Image 2023-03-06 at 20 38 17](https://user-images.githubusercontent.com/1646078/223228972-dac892c7-4886-4b76-950b-d1f6d846cea2.jpg)

**Reply by 1coderookie** [reply]
Thanks!
